src/components/ActionBar/index.tsx

```diff
@@ -26,18 +26,18 @@ const StyledBox = styled(Box)<{ rank: number }>`
 `
 
 type ActionBarProps = {
-  children: React.ReactNode
+  children: ReactNode
```

Comment:
```suggestion
```
`FunctionComponent` (or it's alias `FC`) implies `children`: no need to specify it in Props type definition

Reply:
There is a bug in react types which currently force us to specify children in props

I submitted a PR to correct this https://github.com/DefinitelyTyped/DefinitelyTyped/pull/54955 :)

Reply:
I'm not gonna have it ready anytime soon, I think we'll have to move forward without it for now 😢